app/src/main/kotlin/org/eclipse/kuksa/testapp/databroker/view/DataBrokerConnectionView.kt

```diff
@@ -71,7 +71,9 @@ fun DataBrokerConnection(viewModel: ConnectionViewModel) {
         mutableStateOf(connectionInfoState.value)
     }
 
-    Headline("Connection")
+    if (!viewModel.isConnected) {
```

Comment:
The animation where the Connection title + button is disappearing looks a bit clunky. I think it is because the Title + button are animated separately instead of the whole block. You could wrap the whole block here instead of just the Headline. Maybe also add AnimatedVisibility here so it looks a bit smoother